sct-commons/src/main/java/org/lfenergy/compas/sct/commons/dto/SclReportItem.java

```diff
@@ -0,0 +1,33 @@
+// SPDX-FileCopyrightText: 2022 RTE FRANCE
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package org.lfenergy.compas.sct.commons.dto;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+
+@Getter
+@ToString
+@EqualsAndHashCode
+public final class SclReportItem {
+    private final String xpath;
+    private final String message;
+    private final boolean isFatal;
```

Comment:
is warning used?

Reply:
The issue requires to report non-blocking errors (that is why i created "isFatal').
The warning is used in InputsAdapter.java to report non fatal erros.